src/Messages.sol

```diff
@@ -746,15 +746,15 @@ library ConnectorMessages {
         remainingRedeemOrder = uint128(_msg.indexUint(105, 16));
     }
 
-    function formatAddAdmin(address user) internal pure returns (bytes memory) {
-        return abi.encodePacked(uint8(Call.AddAdmin), user);
+    function formatScheduleUpgrade(address user) internal pure returns (bytes memory) {
```

Comment:
It would be cool to rename user to contract here in solidity as well 🙏🏻

Reply:
`contract` is a keyword in solidity so that won't work, but how about `upgrade`?

Reply:
I suppose `_contract` would be the most appropriate here